helpers/create_application.py

```diff
@@ -14,7 +13,12 @@
 event_type = "et1"
 event_type_display_name = "First Event Type"
 
+f = open("/Users/hrupp/insights/policies-ui-backend/server/src/test/resources/rhid_heiko.txt", "r")
```

Comment:
Could you include a sample rhid without the absolute path? Maybe one that could be used if RBAC is set to off (-Drbac.enabled=false)

Reply:
Also, maybe these changes got committed unintentionally? This is `create_application.py` and the rhid is used to create endpoints.

Reply:
Well. That is "end2end" so the functionality is intentional, but the naming of the file is meanwhile no longer good enough.